github/reactions.go

```diff
@@ -352,6 +394,24 @@ func (s *ReactionsService) CreateTeamDiscussionReaction(ctx context.Context, tea
 	return m, resp, nil
 }
 
+// DeleteTeamDiscussionReaction deletes the reaction to a team discussion.
+//
+// GitHub API docs: https://developer.github.com/v3/reactions/#delete-team-discussion-reaction
+func (s *ReactionsService) DeleteTeamDiscussionReaction(ctx context.Context, org, teamSlug string, discussionNumber, reactionID int64) (*Response, error) {
+	url := fmt.Sprintf("orgs/%v/teams/%v/discussions/%v/reactions/%v", org, teamSlug, discussionNumber, reactionID)
+
+	return s.deleteReaction(ctx, url)
+}
+
+// DeleteTeamDiscussionReactionByTeamIDAndOrgID deletes the reaction to a team discussion by organization ID and team ID.
+//
+// GitHub API docs: https://developer.github.com/v3/reactions/#delete-team-discussion-reaction
+func (s *ReactionsService) DeleteTeamDiscussionReactionByTeamIDAndOrgID(ctx context.Context, orgID, teamID, discussionNumber, reactionID int64) (*Response, error) {
```

Comment:
The method is described as "ByTeamIDAndOrgID", but the parameters are in the opposite order: orgID first, teamID second.

Having orgID be first makes sense, given it's a higher level piece of information. It's consistent with specifying repoID first, issueID second.

What do you think about renaming this method (and others like it) to be consistent with the parameters, i.e., "ByOrgIDAndTeamID"?

(Optional comment.)

Reply:
Another idea here is to just use a simpler "ByID" suffix for all these methods that use IDs instead of repo/org names.

That is what's used by many other existing endpoints:

```
$ go doc -all github.com/google/go-github/github | grep ByID | wc -l    
      55
```

Reply:
I agree, consistency would be good.